### PR TITLE
[s2s] create doc for pegasus/fsmt replication

### PR DESCRIPTION
**Pull request by stas00**
This PR:

* created a dedicated doc for getting eval data
* move the existing entries to the new doc
* add FSMT
* add pegasus

@sshleifer

### CODE REVIEW

File: examples/seq2seq/README.md [hunk 1]
```diff
@@ -15,55 +15,7 @@ For `bertabs` instructions, see [`bertabs/README.md`](bertabs/README.md).
...
-train.source
-train.target
-val.source
-val.target
-test.source
-test.target
-```
-The `.source` files are the input, the `.target` files are the desired output.
-
+See [Getting evaluation data](./eval_datasets.md).
```
Comment (sshleifer) [inline examples/seq2seq/README.md:18]: (1) Never mind my copy paste comment about the original contents of this Section. They should stay here.  This hyperlink should just say, roughly 
> To replicate FSMT or Pegasus, see `.eval_datasets.md`.

(2) Consider separating eval_fsmt.md and replicate_pegasus.md, they seem like very different use cases.

(3) Alternatively, you could try to think about how this whole system should work and try to make documentation easy to navigate for different use cases -- (evaluating a model), (training a model), replicating a paper. If you go this route, keep all the stuff about using your own data high in the page, as this is a large fraction of the training use cases.

Comment (stas00) [reply examples/seq2seq/README.md:18]: Please clarify number 1? you mean to leave everything as is, right? How about I just add pegasus and fsmt notes to README.md and we are done?

I'd be happy to write better docs once I work on such project. I can't do it in void.

Comment (sshleifer) [reply examples/seq2seq/README.md:18]: > How about I just add pegasus and fsmt notes to README.md and we are done?

Yes. But then exit the void :), and make sure that when you read the document as a whole, it coheres.

Comment (sshleifer) [reply examples/seq2seq/README.md:18]: Slack if doesn't make sense, I'm just trying to (a) make your doc discoverable without (b) making existing stuff less clear.

Comment (stas00) [reply examples/seq2seq/README.md]: I added the information in a very succinct way - so it shouldn't interfere with the current doc.

**Review by sshleifer — APPROVED** [review]
thx!

**Pull request merged**